samples/java/testng-selenium/sample_reports/TestRail suite/TestRail Tests .html

```diff
@@ -0,0 +1,145 @@
+<html>
```

Comment:
Can we remove everything under the TestRail suite folder? Or do these have any purpose?